modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderItemLocalServiceImpl.java

```diff
@@ -2641,6 +2642,51 @@ private void _validate(
 		}
 	}
 
+	private void _validateBundleInformation(
+			CPDefinitionOptionRel cpDefinitionOptionRel,
+			List<CPDefinitionOptionValueRel> cpDefinitionOptionValueRels,
+			JSONArray optionJSONArray)
+		throws PortalException {
+
+		for (CPDefinitionOptionValueRel cpDefinitionOptionValueRel :
+				cpDefinitionOptionValueRels) {
+
+			String linkedCPInstanceUUID =
+				cpDefinitionOptionValueRel.getCPInstanceUuid();
+
+			if (Validator.isNotNull(linkedCPInstanceUUID)) {
+				CPInstance linkedCPInstance =
+					_cpInstanceLocalService.getCPInstanceByUuidAndGroupId(
+						linkedCPInstanceUUID,
+						cpDefinitionOptionValueRel.getGroupId());
+
+				for (int i = 0; i < optionJSONArray.length(); i++) {
+					JSONObject optionJSONObject = optionJSONArray.getJSONObject(
```

Comment:
We only validate existing JSON for 2 reasons:
1. We can't assume a specific value (so no 'value' is input)
2. We can't assume a choice (so we only want to check if they've already added it based on key)

Since these options will never be 'required' (the 'required' and 'sku contributor' ones are already covered in earlier logic), we know that we don't necessarily need the information.